<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="module_to_nv_button">
         <property name="text">
          <string>Inverter To NV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="write_to_module_button">
         <property name="text">
          <string>To Inverter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="write_to_file_button">
         <property name="text">
          <string>To File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="write_to_value_set_file_button">
         <property name="text">
          <string>To Value Set File</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="read_from_value_set_file_button">
         <property name="text">
          <string>From Value Set File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read_from_file_button">
         <property name="text">
          <string>From File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read_from_module_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>From Inverter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="Enum" name="current_access_level">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enforce_range_limits_check_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Enforce Range Limits</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="SearchBox" name="searchbox"/>
     </item>
     <item>
      <widget class="QTreeView" name="tree_view">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchBox</class>
   <extends>QWidget</extends>
   <header>epyqlib.searchbox</header>
  </customwidget>
  <customwidget>
   <class>Enum</class>
   <extends>QWidget</extends>
   <header>epyqlib.widgets.enum</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
